src/scripts/main.js

```diff
@@ -357,4 +357,47 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
```

Comment:
table still not rendered in tbody.
try to query selector `tbody` instead `.dashboard`
![Screenshot 2024-09-11 at 21 56 54](https://github.com/user-attachments/assets/92b64957-9838-4678-a92d-4efebf425bc6)